packages/core/src/shared/telemetry/telemetryService.ts

```diff
@@ -116,7 +117,10 @@ export class DefaultTelemetryService {
             })
 
             try {
-                await fs.writeFile(this.persistFilePath, JSON.stringify(this._eventQueue))
+                /**
+                 * This function runs in deactivate() so we must use node fs. See the vscode behavior doc for more info.
+                 */
+                await fsNode.writeFile(this.persistFilePath, JSON.stringify(this._eventQueue))
```

Comment:
I guess in vscode webs case this doesn't matter since it has nothing to persist to anyways. Will it throw an error or anything since we're using fs directly or is that properly stubbed out?